rocketpy/rocket/aero_surface.py

```diff
@@ -2023,6 +2025,12 @@ def deployment_level(self, value):
                 )
         self._deployment_level = value
 
+    def _reset(self):
+        """Resets the air brakes to their initial state. This is ran at the
```

Comment:
Why did you make this method (weakly) private? 
Are you concerned that we eventully introduces breaking changes here? Or do you want to provent users from invoking this method (ever)?

If the answer is the second option, I would suggest to put a second leading underscore to the name of the method.

Reply:
Yes, I was trying to avoid breaking changes. I did not want to make it double score because it gets used in other classes.
If you think double is better I could change it no problem.